src/constants.mjs

```diff
@@ -1,2 +1,2 @@
-export const datocmsEnvironment = 'tags-fresh';
+export const datocmsEnvironment = 'blog-topic-meta-data';
```

Comment:
Is there any migration related to this environment? What do you think of generating them just for the sake of having them in the repo?

Reply:
yes but I need to clean up some content and create a fresh fork after testing. There are some nonsense blogs in tags for testing purposes